README.md

```diff
@@ -33,15 +33,16 @@ gelf.log('server is DOWN')
 gelf.log('server is DOWN', source='hostchecker')
 
 # Send different data fields
-gelf.log('status change', state='DOWN', server='macbook', source='hostchecker')
+gelf.log('status change', state='DOWN', server='macbook', source='hostchecker', level=LogLevel.error)
 
 
 # You can also prepare all data into a dictionary and give that to .log
 data = {}
 data['short_message'] = 'hello from python'
 data['host'] = 'hostchecker'
+data['level'] = LogLevel.alert
```

Comment:
This seems like a redundant value for the example since the default is already LogLevel.alert